source/PlayerInfo.cpp

```diff
@@ -447,6 +447,10 @@ void PlayerInfo::Save() const
 	}
 
 	Save(filePath);
+
+	// Save global Conditions:
+	DataWriter globalConditions(Files::Config() + "global conditions.txt");
+	GameData::GlobalConditions().Save(globalConditions, "global conditions");
```

Comment:
Don't need this anymore.
```suggestion
	GameData::GlobalConditions().Save(globalConditions);
```